migrations/2_deploy_contracts.js

```diff
@@ -28,7 +28,7 @@ module.exports = (deployer, network, accounts) => {
     );
   }
 
-  if (network == "kovan") {
+  if (network === 'kovan') {
     deployer.deploy([
       Exchange,
       // Redeem,
```

Comment:
There was a conflict in this file, where I kept the [most recent change](https://github.com/melonproject/protocol/blob/master/migrations/2_deploy_contracts.js) instead of this one.

I think that is what you want, but let me know if it is the opposite 😃